content/naorlivne/drone-metronome/index.md

```diff
@@ -0,0 +1,76 @@
+---
+version: '1.0'
```

Comment:
You can drop that, we just had version 0.8 to show this is not a 1.0 configuration.

Reply:
Dropped